### ISSUE TITLE
Money reward issue!

### ISSUE THREAD

**Issue by JackkGC** [NoChanceSD/PvPManager#72]
I have the problem that my normal users dont get money for an kill. Only the admins with op perms get the reward!

**Comment by NoChanceSD** [comment]
Are you sure? I just tested with no special perms and got the money.

**Comment by JackkGC** [comment]
Am 28.10.2015 um 00:27 schrieb NoChanceSD:
>
> Are you sure? I just tested with no special perms and got the money.
>
> —
> Reply to this email directly or view it on GitHub 
> <https://github.com/NoChanceSD/PvPManager/issues/72#issuecomment-151674213>.
>
Im sorry that I wrote the ticket so fast without testing another economy 
plugin. It was an issue with Vault an Fe-Economy. Vault is not 
compatible with it so it didnt worked well all the time. I thought it 
was your plugin because it worked almost with every other plugin wich 
handles money. Now I got an other solution. But anyway thanks for 
answering :D
